crates/evm/src/tests/sys_tx_tests.rs

```diff
@@ -416,7 +415,7 @@ fn test_bridge() {
     let (mut config, _, _) =
         get_evm_config_starting_base_fee(U256::from_str("1000000").unwrap(), None, 1);
 
-    config_push_contracts(&mut config);
+    config_push_contracts(&mut config, None);
```

Comment:
why not call `config_push_contracts` inside `get_evm_*` functions?

Reply:
makes sense